pandas/tests/io/json/test_normalize.py

```diff
@@ -561,6 +561,14 @@ def test_top_column_with_leading_underscore(self):
 
         tm.assert_frame_equal(result, expected)
 
+    def test_series_index(self, state_data):
+        idx = [7, 8]
+        series = Series(state_data, index=idx)
+        result = json_normalize(series)
+        assert (result.index == idx).all()
```

Comment:
Could you make `idx` a `pandas.Index` and then use 

`tm.assert_index_equal(result.index, idx)`